sentinel_alert_rule/locals.tf

```diff
@@ -0,0 +1,8 @@
+
+# locals {
+#   common_tags = {
+#     (var.billing_tag_key) = var.billing_tag_value
+#     Terraform             = "true"
+#   }
+# }
```

Comment:
Can we add tags to some of those resources, if not then we should just delete this file.

Reply:
The tags will be added automatically by Azure Landing Zone.  I'll delete this file.